{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py

```diff
@@ -0,0 +1,46 @@
+"""
+Configuration items for server startup.
+"""
+import os
+
+
+class BaseConfig(object):
+    """
+    Base configuration.
+    """
+    V0_1_PATH = '/{{ cookiecutter.project_slug }}/api/v0.1'
+
+    #
+    # Debug toolbar
+    #
+    DEBUG_TB_ENABLED = False
+    DEBUG_TB_INTERCEPT_REDIRECTS = False
+
+    #
+    # Flask-Restplus
+    #
+    RESTPLUS_SWAGGER_UI_DOC_EXPANSION = 'list'
+    RESTPLUS_VALIDATE = True
+    RESTPLUS_MASK_SWAGGER = False
+    RESTPLUS_ERROR_404_HELP = False
+
+
+class DevConfig(BaseConfig):
+    """
+    Development configuration.
+    """
+    ENV = 'dev'
+    DEBUG = True
+
+    #
+    # Flask debug toolbar
+    #
+    DEBUG_TB_ENABLED = True
+
+
+#
+# Set up config based on environment
+#
+ENV = os.environ.get('ENV')
```

Comment:
:} the infection is spreading

Reply:
You don't like this? I'd prefer to work a better config solution into this, but I don't know what that is.

Reply:
I don't mind it as much as other folks on the team because it gets the job done. Could it be done with a flask feature or abstracted out of the config module? Ideally, we sort that out in this project.

Reply:
Let's re-visit once we've finalized how envkey will work.